Gem/Code/Source/ProceduralSkinnedMeshUtils.h

```diff
@@ -28,7 +28,7 @@ namespace AtomSampleViewer
 {
     class ProceduralSkinnedMesh;
 
-    AZ::Data::Instance<AZ::RPI::Model> CreateModelFromProceduralSkinnedMesh(const ProceduralSkinnedMesh& proceduralMesh);
-    AZStd::intrusive_ptr<AZ::Render::SkinnedMeshInputBuffers> CreateSkinnedMeshInputBuffersFromProceduralSkinnedMesh(const ProceduralSkinnedMesh& proceduralMesh);
+    AZ::Data::Instance<AZ::RPI::Model> CreateModelFromProceduralSkinnedMesh(ProceduralSkinnedMesh& proceduralMesh);
```

Comment:
Feel like these functions deserve a comment indicating what happens to the input parameters.